client/landing/stepper/declarative-flow/internals/steps-repository/goals/style.scss

```diff
@@ -61,6 +61,10 @@
 			width: 130px;
 			height: 44px;
 			border-radius: 4px;
+
+			&:focus {
+				box-shadow: 0 0 0 2px #fff, 0 0 0 4px var( --color-accent-60 );
```

Comment:
Maybe we could consider using the same `box-shadow` as other steps in the flow use:

```suggestion
				box-shadow: 0 1px 2px rgb( 0 0 0 / 5% );
```

This would be the result (gif):
![Screen Capture on 2022-06-24 at 10-13-18](https://user-images.githubusercontent.com/25105483/175493536-c6fcaeea-7a47-4ce6-8755-276b3a417d57.gif)

Reply:
I do not like the fact that this is not implemented in the primary button itself in the `@automattic/components`. Not sure what is the process of contributing into the components package.

As a quick fix, as @ivan-ottinger already pointed out, it probably makes sense to copy one of the current implementations of this from another step. There is one for example in `.podcast-title__submit-button` in `client/landing/stepper/declarative-flow/internals/steps-repository/podcast-title/style.scss`.
![Screenshot 2022-06-24 at 10 26 56](https://user-images.githubusercontent.com/2019970/175496951-d2421489-cc3c-469e-9b19-12d12d669df6.png)
![Screenshot 2022-06-24 at 10 27 08](https://user-images.githubusercontent.com/2019970/175496954-7de38f8b-7d1e-437d-b9dd-a65e98038d8b.png)

Reply:
Yeah, it looks like each step has slightly different implementation, but at the end, the `box-shadow` is set to `0 1px 2px rgba( 0, 0, 0, 0.05 );`.

Example from `client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/form.scss`:
```
.button.site-vertical__submit-button.is-primary {
	padding: 9px 48px;
	border-radius: 4px;
	font-weight: 500;

	// override unnecessary super specificity added by another class
	box-shadow: 0 1px 2px rgba( 0, 0, 0, 0.05 ) !important;

	&:disabled {
		color: #fff;
		background-color: #dcdcde;
	}
}
```

Reply:
But for consistency we can follow the other steps implementation, just changing the background while focused

Reply:
Fixed!

Reply:
> FYI, I used the wordpress.com implementation as a visual reference

🤔 Interesting. It looks like we use a couple of different styles for buttons across the platform.